lib/controllers/login.js

```diff
@@ -10,6 +10,7 @@ var forms = require('../forms');
 var helpers = require('../helpers');
 var oauth = require('../oauth');
 var loginWithOAuthProvider = require('../helpers/login-with-oauth-provider');
+var getLoginViewModel = require('../helpers/get-login-view-model');
```

Comment:
Since `helpers` is already included, if `get-login-view-model` is in `helpers/index.js` you don't have to include it again.